resources/assets/components/utilities/ScholarshipCard/ScholarshipCard.js

```diff
@@ -49,6 +53,14 @@ const ScholarshipCard = ({ campaign }) => {
     { height: 820, width: 1460 },
   ]);
 
+  const handleScholarshipCardShareClick = () => {
+    trackAnalyticsEvent('clicked_scholarshipCard_action', {
```

Comment:
It looks like the event name in the ticket is `phoenix_clicked_scholarship_gallery_block_apply_now` which means here, we want to paste that event name exactly, but we could just strip off the `phoenix` bit since that's added automatically:
```suggestion
    trackAnalyticsEvent('clicked_scholarship_gallery_block_apply_now', {
```